service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/cmo/AgreedCMOUploadedContentProvider.java

```diff
@@ -27,13 +27,15 @@ public CMOReadyToSealTemplate buildTemplate(HearingBooking hearing, Long caseId,
             .judgeName(getJudgeName((judge)))
             .judgeTitle(getJudgeTitle((judge)))
             .respondentLastName(getFirstRespondentLastName(respondents))
-            .subjectLineWithHearingDate(subjectLine(hearing, respondents, familyManCaseNumber))
+            .subjectLineWithHearingDate(subject(hearing, respondents, familyManCaseNumber))
             .build();
     }
 
-    private String subjectLine(HearingBooking hearing, List<Element<Respondent>> respondents,
-                               String familyManCaseNumber) {
-        return String.format("%s, %s", buildSubjectLine(familyManCaseNumber, respondents),
-            uncapitalize(hearing.toLabel()));
+    private String subject(HearingBooking hearing, List<Element<Respondent>> respondents, String familyManCaseNumber) {
+        String subject = buildSubjectLine(familyManCaseNumber, respondents);
+        if (hearing == null) {
```

Comment:
if hearing is not present for Agreed CMO then it is an error i guess, cannot have Agreed CMO without hearing

Reply:
added an exception